### PR TITLE
Added writeJsonObject method for write JsonSerializable object

### PR DESCRIPTION
**Pull request by SiViN**

### CODE REVIEW

File: tests/cases/Psr7Response.phpt [hunk 1]
```diff
@@ -7,13 +7,15 @@
...
 use Contributte\Psr7\Psr7ResponseFactory;
+use Contributte\Psr7\Tests\Helpers\JsonObject;
 use Nette\Application\IResponse as IApplicationResponse;
 use Nette\Application\Responses\TextResponse;
 use Nette\Http\IResponse;
 use Nette\Http\Response;
 use Tester\Assert;
 
 require_once __DIR__ . '/../bootstrap.php';
+require_once __DIR__ . '/../helpers/JsonObject.php';
```
Comment (f3l1x) [inline tests/cases/Psr7Response.phpt]: Please, move it to /tests/fixtures and setup autoload-dev. :-)

File: src/Extra/ExtraResponseTrait.php [hunk 2]
```diff
@@ -54,6 +54,17 @@ public function writeJsonBody(array $data)
 			->withHeader('Content-Type', 'application/json');
 	}
 
+	/**
+	 * @param \JsonSerializable $object
```
Comment (f3l1x) [inline src/Extra/ExtraResponseTrait.php]: Import this class -> `JsonSerializable`

**Review by f3l1x — CHANGES_REQUESTED** [review]

**Comment by f3l1x** [comment]
Good job, thank you. A few tiny details, I will merge it after that.

File: tests/cases/Psr7Response.phpt (hunk 1 above)
Comment (SiViN) [reply tests/cases/Psr7Response.phpt]: Check last travis build, autoload-dev not working in travis..nette tester on localhost is fine

**Pull request merged**